core/src/error.rs

```diff
@@ -126,6 +126,20 @@ impl fmt::Display for SchemeError {
     }
 }
 
+#[derive(Debug, PartialEq, Clone, Copy, Eq)]
+pub enum TransactionError {
```

Comment:
For the `Error` part, we already have `KeyError::InvalidSignature`. Let's use it instead.

Reply:
I removed `TransactionError`. Thank you!